x/mint/types/params.go

```diff
@@ -0,0 +1,94 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store keys
+var (
+	KeyMintDenom     = []byte("MintDenom")
+	KeyBlocksPerYear = []byte("BlocksPerYear")
+)
+
+// ParamTable for minting module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(
+	mintDenom string, inflationRateChange, inflationMax, inflationMin, goalBonded sdk.Dec, blocksPerYear uint64,
+) Params {
+	return Params{
+		MintDenom:     mintDenom,
+		BlocksPerYear: blocksPerYear,
```

Comment:
Need to think about that one. On one hand, I think the gov-surface should be as small as possible and that would also apply here. On the other hand, the block times are not very stable and it might be good to leave this changeable without requiring a software upgrade.

Reply:
Removed this param.